internal/driver/driver_test.go

```diff
@@ -203,6 +203,57 @@ func TestParse(t *testing.T) {
 	}
 }
 
+func TestAppendComment(t *testing.T) {
+
+	// Encode profile into a protobuf and decode it again.
+	protoTempFile, err := ioutil.TempFile("", "profile_proto")
+	if err != nil {
+		t.Errorf("cannot create tempfile: %v", err)
+	}
+	defer os.Remove(protoTempFile.Name())
+	defer protoTempFile.Close()
+
+	testFlagSet := baseFlags()
+	testFlagSet.bools["proto"] = true
+	testFlagSet.strings["comment"] = "test comment please ignore"
+	testFlagSet.strings["output"] = protoTempFile.Name()
+	testFlagSet.args = []string{"cpu"}
```

Comment:
Consider using "cpusmall" instead of "cpu" since profile content isn't super important here, and it might be easier to debug this test case with a smaller profile?

Reply:
done